export const handleException = (status) => {
  console.log(status);
};
